net-dns/dnsviz/dnsviz-0.8.2.ebuild

```diff
@@ -0,0 +1,36 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python{2_7,3_4,3_5,3_6} )
+inherit distutils-r1
+
+MY_PN="dnsviz"
+
+DESCRIPTION="Tool suite for analysis and visualization of DNS and DNSSEC"
+HOMEPAGE="http://dnsviz.net/"
+SRC_URI="https://github.com/dnsviz/${MY_PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+SLOT="0"
+LICENSE="GPL-2"
+KEYWORDS="~amd64 ~x86"
+IUSE="bind"
```

Comment:
ypu do not need this flag...

Reply:
I see. 
@juippis gived me a nice feedback.Removed.